iml-agent/src/cli.rs

```diff
@@ -415,12 +430,23 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             }
             Err(e) => println!("{:?}", e),
         },
-        App::PackageInstalled { package } => {
-            match package_installed::package_installed(package).await {
-                Ok(cs) => {
-                    println!("{}", if cs { "Installed" } else { "Not Installed" });
+        App::Package { command } => {
+            if let Err(e) = match command {
+                PackageCommand::Installed { package_name } => package::installed(package_name)
+                    .await
+                    .map(|r| println!("{}", if r { "Installed" } else { "Not Installed" })),
+                PackageCommand::Version { package_name } => {
+                    package::version(package_name).await.map(|r| match r {
+                        package::RpmResult::Ok(v) => println!("{}", v),
+                        package::RpmResult::Err(e) => {
+                            println!("{}", e);
```

Comment:
Maybe this better print to stderr?

Reply:
`tracing::error!()` is probably best.

Reply:
Sure.

Bear in mind OST pools commands do print errors to stdout though.

`tracing::error!()` is more of a logging solution, I don't think we need it here.